vassal-app/src/main/java/VASSAL/build/module/map/Zoomer.java

```diff
@@ -660,8 +666,36 @@ public void addTo(Buildable b) {
     map.getToolBar().add(zoomInButton);
     map.getToolBar().add(zoomPickButton);
     map.getToolBar().add(zoomOutButton);
-  }
 
+    // Ctrl+Mousewheel to zoom in/out.
+    listener = new MouseWheelListener() {
```

Comment:
This could be a lambda, as `MouseWheelListener` has only one method.

Reply:
I'm happy to turn it into one if you supply the syntax. I will confess to having googled for 10 minutes and can't figure it out. Lambdas for me make the code take up fewer lines at the expense of rendering it incomprehensible. I do need "listener" to be set for later, if that matters.

Reply:
It would look like this:
```
      listener = e -> {
        if (e.getScrollAmount() == 0) {
          return;
        }

        if ((e.getScrollType() == MouseWheelEvent.WHEEL_UNIT_SCROLL) && SwingUtils.isSelectionToggle(e)) {
          int units = e.getUnitsToScroll();

          if ((units < 0) && state.hasHigherLevel()) {
            zoomIn();
          }
          else if ((units > 0) && state.hasLowerLevel()) {
            zoomOut();
          }
        }

        map.getComponent().getParent().dispatchEvent(e); // So that the scrollbars can still find our event.
      };

Reply:
done